apps/docs/content/dotnet/how-to/customize-runtime.mdx

```diff
@@ -16,39 +16,45 @@ import Image from '/src/components/Image';
+
+Additional packages and tools can be installed using [run.prepareCommands](/references/zeropsyml#preparecommands-optional-1).
+:::
 
-If you prefer the Ubuntu OS instead of Alpine, set the `<a href="/dotnet/how-to/build-pipeline#os">build.os</a>` attribute to `ubuntu`. To install additional packages or tools add one or more <a href="/dotnet/how-to/build-pipeline#preparecommands-1">`run.prepareCommands`</a> commands to your `zerops.yml`.
+## Runtime Flow 
 
 When the first deploy with a defined `prepareCommands` attribute is triggered, Zerops will
 
-1. create a prepare runtime container
-2. optionally: [copy selected folders or files from your build container](/dotnet/how-to/build-pipeline#copy-folders-or-files-from-your-build-container)
-3. run the `run.prepareCommands` commands in the defined order
+1. Create a prepare runtime container
+2. Optionally: [copy selected folders or files from your build container](/dotnet/how-to/build-pipeline#copy-folders-or-files-from-your-build-container)
+3. Run the `prepareCommands` in the defined order
 
-### Command exit code
+## Command exit code
 
 If any command fails, it returns an exit code other than 0 and the deploy is canceled. Read the [prepare runtime log](/dotnet/how-to/logs#prepare-runtime-log) to troubleshoot the error. If the command ends successfully, it returns the exit code 0 and Zerops triggers the following command. When all `prepareCommands` commands are finished, your custom runtime environment is ready for the deploy phase.
 
 The prepare runtime container is automatically deleted after the prepare runtime phase has finished or failed.
 
-### Custom runtime environment cache
+## Custom runtime environment cache
 
 Some packages or tools can take a long time to install. Therefore, Zerops caches your custom runtime environment after the installation of your custom packages or tools is completed. When the second or following deploy is triggered, Zerops will use the custom runtime cache from the previous deploy if following conditions are met:
 
-1. Content of the <a href="/dotnet/how-to/build-pipeline#copy-folders-or-files-from-your-build-container">`build.addToRunPrepare`</a> and <a href="/dotnet/how-to/build-pipeline#preparecommands-1">`run.prepareCommands`</a> attributes didn't change from the previous deploy
+1. Content of the <a href="/dotnet/how-to/build-pipeline#copy-folders-or-files-from-your-build-container">build.addToRunPrepare</a> and <a href="/dotnet/how-to/build-pipeline#preparecommands-1">run.prepareCommands</a> attributes didn't change from the previous deploy
 2. The custom runtime cache wasn't invalidated in the Zerops GUI.
 
-To invalidate the custom runtime cache go to `yyy`
+To invalidate the Zerops runtime cache go to your service detail in Zerops GUI, choose **Service dashboard & runtime containers** from the left menu and click on the **Open pipeline detail** button. Then click on the **Clear runtime prepare cache** button.
```

Comment:
what do you think about this layout?

![image](https://github.com/user-attachments/assets/26d7f92f-6a2b-466d-ab29-c423219cd322)